src/frameworks/docpad.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "docpad",
+  "category": "static_site_generator",
+  "detect": {
+    "npmDependencies": ["docpad"],
+    "excludedNpmDependencies": [],
+    "configFiles": []
+  },
+  "watch": {
+    "command": "docpad run",
```

Comment:
Serve installer is broken:
https://github.com/docpad/docpad/issues/1124

Used the docs for the command and port:
https://docpad.bevry.me/start/begin